src/crystal/system/unix/process.cr

```diff
@@ -79,6 +79,7 @@ struct Crystal::System::Process
     if ret == 0
       true
     else
+      return true if Errno.value == Errno::EPERM
```

Comment:
Have you considered rewriting this as a `case Errno.value` statement?
I think it would work very nicely here, being more expressive. And some benefit is calling `Errno.value` only once.

Reply:
makes sense, fixed